mindsdb/integrations/handlers/utilities/query_utilities/update_query_utilities.py

```diff
@@ -43,7 +44,7 @@ def parse_set_clause(self):
                 if value[0] not in self.supported_columns:
                     raise UnsupportedColumnException(f"Unsupported column: {value[0]}")
             
-            values_to_update[value[0]] = value[1].value
+            values_to_update[value[0]] = value[1]
```

Comment:
Are you sure that this is correct? I think `.value` needs to be included.

Reply:
I'm a donkey and put trailing commas in the dict leading to the strings becoming tuples. WIll fix.

Reply:
Haha not all! Please don't worry about it.

Reply:
In first line values = list(self.query.update_columns.items()) returns a list of tuples. Tuples don't have .value.